tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -815,40 +748,6 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
         validations=[{"batch_request": batch_request}],
     )
 
-    assert len(context.validations_store.list_keys()) == 1
-    assert result["success"]
-
-
-@pytest.mark.spark
-def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_run_validations_batch_request_dataframe_in_context_run_checkpoint_sparkdf(  # noqa: E501
```

Comment:
Why delete this whole test?

Reply:
Honestly think it was accidental - should also bring back the `result["success"]` assertion above